rest_tools/server/handler.py

```diff
@@ -135,6 +145,59 @@ def write_error(self, status_code=500, **kwargs):
         self.write(data)
         self.finish()
 
+    def get_json_body_argument(
+        self,
+        name: str,
+        default: Any = arghandler.NO_DEFAULT,
+        choices: Optional[List[Any]] = None,
+    ) -> Any:
+        """Return the argument from JSON-decoded request body.
+
+        If no `default` is provided, and the argument is not present, raise `400`.
+
+        Arguments:
+            name -- the argument's name
+
+        Keyword Arguments:
+            default -- a default value to use if the argument is not present
+            choices -- a list of valid argument values (raise `400`, if arg's value is not in list)
+
+        Returns:
+            Any -- the argument's value, unaltered
+        """
+        return arghandler.ArgumentHandler.get_json_body_argument(
+            super(), name, default, choices
```

Comment:
Do you want `super()` here and not `self`?

Seems like it would prevent subclasses from customizing the behavior of RequestHandler, like `get_argument()` for instance.

Maybe that's desired though? You want Tornado's version those methods no matter what?

Reply:
> Maybe that's desired though? You want Tornado's version those methods no matter what?

Exactly. This was my way of adding a encapsulated static class (`ArgumentHandler`) instead of directly placing code in `RestHandler`.